backend/src/test/java/at/spengergasse/backend/persistence/UserRepositoryTest.java

```diff
@@ -0,0 +1,70 @@
+package at.spengergasse.backend.persistence;
+
+import at.spengergasse.backend.model.User;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.sql.Date;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+class UserRepositoryTest {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Test
+    public void testSaveUserAndFindById() {
+        // Erstelle einen neuen Benutzer
+        User user = User.builder()
+                .firstname("Max")
+                .lastname("Mustermann")
+                .email("max.mustermann@example.com")
+                .password("pass123")
+                .deleted(false)
+                .created(new java.sql.Date(System.currentTimeMillis()))
+                .build();
+
+        // Speichere den Benutzer in der Datenbank
+        User savedUser = userRepository.save(user);
+
+        assertNotNull(savedUser);
+
+        // Suche den Benutzer nach der ID
+        User retrievedUser = userRepository.findById(user.getId());
+
+        // Überprüfe, ob der gespeicherte Benutzer korrekt abgerufen wurde
+        assertNotNull(retrievedUser);
+        assertEquals("Max", retrievedUser.getFirstname());
```

Comment:
all these asserEquals are uneccassary and use assertThat().equals...

but ig it is not bad to overtest so all these tests are fine